### ISSUE TITLE
Feature: Custom Design

### ISSUE THREAD

**Issue by fbacker** [xgfe/react-native-datepicker#30]
Possible to use an own design and only use the 'modal' selector or be able to override the design by 

	<DatePicker
	  date={this.state.date}
	  mode="date"
	  onDateChange={(date) => {this.setState({date: date})}}
	>
	  <Text>Adding my custom design, button and icons</Text>
	</DatePicker>

**Comment by feyy** [comment]
You can almost override all styles using `customStyles ` property.

**Comment by aferust** [comment]
+1
as @fbacker has pointed, we cannot display for instance localize date string in built-in text of datepicker component. In this case I don't want to modify format prop, because I need specific format for the data (not the date to be displayed) for mysql storage maybe. For existing datepicker component date-time string to be displayed and format prop must be same. This is a serious problem. I want to display a localize date-time for users, but to store date-time in other format complying with mysql.

it should be desired that:
date-time string to be displayed:
"Aug 16, 2016" (could be in different localization setting such as "16 Ağustos 2016")

date-time string returned by the component:
"2016-08-16 10:30"

**Comment by feyy** [comment]
@aferust Datepicker used [Moment.js](http://momentjs.com/) format the date, so you can use it's localization feature. 
![huaban](https://cloud.githubusercontent.com/assets/3056002/17836064/2e21d60e-67b8-11e6-9a8b-2734e1cee715.png)

**Comment by luisfuertes** [comment]
On Android use native component. Can add the option of use a datePicker similar than iOS?

**Comment by feyy** [comment]
@luisfuertes Yes, we have plans to do that. But waiting for another cross platform picker component.

**Comment by joncursi** [comment]
Would be great if we could render our own input component and then call the opening of this date picker through a method, perhaps referencing it by ref. I love the modal implementation of this component but I want my input component to be totally different to match my app's unique design. Overriding all the styles to transform this component's input into something else isn't always an option, and perhaps you would want to call the opening of the modal from more than one place.

Adding this method would solve all of that!
